src/pages/Tasks/containers/TasksContainer/index.js

```diff
@@ -0,0 +1,70 @@
+import { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+import {
+  completedTask,
+  createTask,
+  editingTask,
+  editingTaskCancel,
+  editingTaskConfirm,
+  removeTask,
+} from "../../actions";
+import Task from "../../components/Task";
+import TasksView from "../../components/TasksView";
+import { tasksSelector } from "../../selectors";
+
+const TasksContainer = () => {
+  const [task, setTask] = useState("");
+  const handleTaskChange = (event) => {
+    setTask(event.target.value);
+  };
```

Comment:
you write this logic twice. create custom hook for handling input storage and use it here and in task component as well